crates/noirc_frontend/src/parser/parser.rs

```diff
@@ -722,20 +722,37 @@ fn assertion<'a, P>(expr_parser: P) -> impl NoirParser<Statement> + 'a
 where
     P: ExprParser + 'a,
 {
-    ignore_then_commit(keyword(Keyword::Assert), parenthesized(expr_parser))
+    let argument_parser =
+        expr_parser.separated_by(just(Token::Comma)).allow_trailing().at_least(1).at_most(2);
+
```

Comment:
Hmm there seems to be duplicated code between assert_eq and assert

Reply:
Yes, but I wouldn't like merging the two parsers just because they share the fact that they look like a function with N or N + 1 args hmmm